README.md

```
@@ -1,43 +1,42 @@
-# ByteStash
+# ClaudeStash
 <p align="center">
   <img src="https://raw.githubusercontent.com/jordan-dalby/ByteStash/refs/heads/main/client/public/logo192.png" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)